sandbox/public/perf-test/legacy-launch.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Legacy - Launch</title>
+
+    <script>
+      (function(win, doc, style, timeout) {
+        var STYLE_ID = "at-body-style";
+
+        function getParent() {
+          return doc.getElementsByTagName("head")[0];
+        }
+
+        function addStyle(parent, id, def) {
+          if (!parent) {
+            return;
+          }
+
+          var style = doc.createElement("style");
+          style.id = id;
+          style.innerHTML = def;
+          parent.appendChild(style);
+        }
+
+        function removeStyle(parent, id) {
+          if (!parent) {
+            return;
+          }
+
+          var style = doc.getElementById(id);
+
+          if (!style) {
+            return;
+          }
+
+          parent.removeChild(style);
+        }
+
+        addStyle(getParent(), STYLE_ID, style);
+        setTimeout(function() {
+          removeStyle(getParent(), STYLE_ID);
+        }, timeout);
+      })(window, document, "body {opacity: 0 !important}", 3000);
+    </script>
+
+    <!-- Using Unified JS LAB, property "alloyio.com [perf w/ old libs]" -->
```

Comment:
In a meeting this morning, Joe said that you had to add the Audience Manager extension in order for SSF to work. Did you try configuring the Analytics extension to use Audience Manager?  There is a section in the Analytics Extension for "Adobe Audience Manager"

Reply:
Yeah, that's what we were trying to get to work but it wouldn't for some reason. That's what prompted us to try to Audience Manager extension--to see if it was just a problem with the Analytics integration. When we unchecked "Automatically share Analytics Data with Audience Manager" in Analytics and then installed the Audience Manager extension, destinations started to work. Maybe you might know why it wasn't working in without the Audience Manager extension? We were thinking it might be appropriate to just use the Audience Manager extension for our performance scenarios anyway because that's what some of our customers use due to other features DIL provides, but I'm a bit clueless in that area.

Reply:
@Aaronius and @jonsnyder was the changes made to use the DIL integration method to call visitor, I have never used the checkbox option but manually works when you call in do plugins when editing library code. From my understanding in Launch you also need the the ECID extension for this to work and the right configuration.

Reply:
There is a setting in the report suite admin pages to enable server side forwarding. This setting must be enabled for server side forwarding to work using the Analytics launch extension option.